e2e-tests/portal/cypress.json

```diff
@@ -15,7 +15,7 @@
   "projectId": "sefntb",
   "testFiles": "**/*.spec.js",
   "pageLoadTimeout": 180000,
-  "numTestsKeptInMemory": 1,
+  "numTestsKeptInMemory": 100,
```

Comment:
not a show stopper, but this shouldn't be pushed to the PR. this is only for testing purposes

Reply:
whoops, will remove

Reply:
removed